avocado/core/loader.py

```diff
@@ -717,7 +719,8 @@ def _make_avocado_tests(self, test_path, make_broken, subtests_filter,
                 else:
                     # Module does not have an avocado test class inside, and
                     # it's not executable. Not a Test.
-                    return make_broken(NotATest, test_path)
+                    return make_broken(NotATest,
```

Comment:
As yet another suggestion, classes such as `NotATest` could, optionally, have a general error message, used if no other was provided.  It seems to match the requirements here, that is "Does not look like an INSTRUMENTED test, nor is it executable" seems closely related to `NotATest`, while "Is not readable" is closely related to `AccessDenied`.